katello_certs/hooks/boot/02-message-helpers.rb

```diff
@@ -1,3 +1,5 @@
+require 'shellwords'
+
```

Comment:
This is no longer needed.
```suggestion
```

Reply:
It does help if you push the most up to date version you have locally to the remote.

Reply:
I've never had that problem **whistle**